src/packaging_dists/_utils.py

```diff
@@ -7,8 +28,11 @@ def _find_name_version_sep(fragment, canonical_name):
     # type: (str, str) -> int
     """Find the separator's index based on the package's canonical name.
 
-    Copied verbatim from pip. This function is needed since the canonicalized
-    name does not necessarily have the same length as the egg info's name part.
+    Copied verbatim from:
+      pip._internal.index.package_finder._find_name_version_sep
```

Comment:
Could you change this (and the one below) to use rst code blocks?

```
Copied verbatim from::

    pip._internal.index.package_finder._find_name_version_sep
```

Reply:
Done!